#DetectionArray.msg

std_msgs/Header header
Detection[] detections